Sources/UID2/UID2Manager.swift

```diff
@@ -167,18 +201,29 @@ public final actor UID2Manager {
         // Process Opt Out
         if let status = status, status == .optOut {
             self.identity = nil
-            KeychainManager.shared.deleteIdentityFromKeychain()
             self.identityStatus = .optOut
+            let identityPackageOptOut = IdentityPackage(valid: false, errorMessage: "User Opted Out", identity: nil, status: .optOut)
+            KeychainManager.shared.deleteIdentityFromKeychain()
```

Comment:
Is this `delete` really required if you're saving a new package immediately? The reason I ask is that in other places where you're updating the Identity and calling `saveIdentityToKeychain`, I don't see an explicit delete.

Reply:
It's probably not required, but since this is a special case as it's for `optOut` I wanted to be explicit about the UID2 Identity being cleared from both memory and disk.